api/src/transactions.rs

```diff
@@ -73,6 +77,20 @@ pub fn submit_json_transactions(context: Context) -> BoxedFilter<(impl Reply,)>
         .boxed()
 }
 
+// POST /transactions/simulate with JSON
+pub fn simulate_json_transactions(context: Context) -> BoxedFilter<(impl Reply,)> {
+    warp::path!("transactions" / "simulate")
+        .and(warp::post())
+        .and(warp::body::content_length_limit(
+            context.content_length_limit(),
+        ))
+        .and(warp::body::json::<UserTransactionRequest>())
```

Comment:
UserCreateSigningMessageRequest
maybe?

Reply:
Are we trying to eliminate the existence of a signature at all? If that's the case we need to create a new message type specifically for simulation, wihch looks like `SubmitUserTransactionRequest` with all signature fields explicitly dropped. We still need their pubkeys (checked in prologue to ensure they match auth key), that's why `UserCreateSigningMessageRequest` with signature field dropped may not work.

Reply:
Hmm... that's true and then we have MultiAgent transactions... :exploding_head: